package.json

```diff
@@ -91,27 +90,33 @@
     "webpack-sources": "^1.1.0"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.19",
+    "@babel/preset-react": "^7.18",
+    "@babel/runtime": "^7.19",
+    "@sect/modernizr-loader": "^1.0.2",
     "@silverstripe/eslint-config": "^0.1.0",
-    "@silverstripe/webpack-config": "^1.7.0",
-    "@storybook/addon-actions": "^3.4.10",
-    "@storybook/addon-knobs": "^3.4.10",
-    "@storybook/addon-notes": "^3.4.10",
-    "@storybook/addon-options": "^3.4.10",
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "babel-jest": "^23.6.0",
-    "babel-runtime": "^6.26.0",
+    "@silverstripe/webpack-config": "file:../webpack-config",
```

Comment:
That bit here assumes that `webpack-config` is one folder up from `admin`.

That's just a temporary hack.